src/main/java/com/google/devtools/build/lib/skyframe/SkyframeExecutor.java

```diff
@@ -3471,9 +3473,28 @@ protected WorkspaceInfoFromDiff handleDiffs(
     Set<Pair<Root, ProcessableModifiedFileSet>> pathEntriesWithoutDiffInformation =
         Sets.newHashSet();
     ImmutableList<Root> pkgRoots = pkgLocator.get().getPathEntries();
+
+    Path workspacePath = directories.getWorkspace();
+    EvaluationResult<SkyValue> evaluationResult =
+      evaluateSkyKeys(eventHandler, ImmutableList.of(IgnoredPackagePrefixesValue.key()), false);
```

Comment:
nit: Perhaps you can make `SkyframeExecutor#evaluateSkyKeys( eventHandler, skyKeys)` package-private and use that instead.

Reply:
Er, I don't think I need to change the visibility? Just dropping the third param works fine. Thanks for catching.